arp.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2018 The Prometheus Authors
+// Copyright 2018 Sam Kottler <shkottler@gmail.com>
+
```

Comment:
Please keep the license as one contiguous block and prefix this empty line with `//`.